src/components/promo.tsx

```diff
@@ -56,6 +56,21 @@ export const Promo = () => {
       link: 'https://pris.ly/sidebar-promo/serverless-architecture-global-scaling',
       color: 'indigo',
     },
+    {
+      text: `Effortlessly uncover and fix slow SQL queries in your app.`,
```

Comment:
```suggestion
      text: `Easily identify and fix slow SQL queries in your app.`,
```

I'd like to opt for simpler language. "Easy" is clear, and I think comes across less "fluffy".
"Identify" I think is more tangible than "uncover".